duo_client/admin.py

```diff
@@ -519,6 +527,95 @@ def get_authentication_log(self, api_version=1, **kwargs):
                 row['host'] = self.host
         return response
 
+    def get_activity_logs(self, **kwargs):
+        """
+        Returns activity log events.
+
+        As of now, the activity endpoint is not in general availability and is restricted to a few customers for private preview.
+        If you have any questions or need more information, feel free to reach out to support for guidance.
+
+
+        mintime (required) - Unix timestamp in ms; fetch records >= mintime
+        maxtime (required) - Unix timestamp in ms; fetch records <= maxtime
+        limit - Number of results to limit to
+        next_offset - Used to grab the next set of results from a previous response
+        sort - Sort order to be applied
+
+        Returns:
+            {
+                "items": [
+                    {
+                        "access_device": {
+                            "browser": <str:browser>,
+                            "browser_version": <str:browser version>,
+                            "ip": {
+                                "address":  <str:ip address>
+                            },
+                            "location": {
+                                "city":  <str:city>,
+                                "country":  <str:country>,
+                                "state": <str:state>
+                            },
+                            "os":  <str:os name>,
+                            "os_version":  <str:os_version>
+                        },
+                        "action":  <str:action>,
+                        "activity_id":  <str:activity id>,
+                        "actor": {
+                            "details": <str:json for actor details>
+                            "key" :  <str:actor's key>,
+                            "name":  <str:actor's name>,
+                            "type":  <str:actor type>
+                        },
+                        "akey":  <str:akey>,
+                        "application": {
+                            "key":  <str:application's key>,
+                            "name":  <str:application's name>,
+                            "type":  <str:application's type>
+                        },
+                        "target": {
+                            "details":  <str:target's details if available> ,
+                            "key" :  <str:target's key>,
+                            "name":  <str:target's name>,
+                            "type":  <str:target's type>
+                        },
+                        "ts":  <str:timestamp captured for activity>
+                    },
+                ],
+                "metadata": {
+                    "next_offset": [
```

Comment:
The type of this example should be a single string per a recent update, as a comma separated entry. i.e `"next_offset": "1532951895000,af0ba235-0b33-23c8-bc23-a31aa0231de8"